UnityPackage/config

```diff
@@ -1,4 +1,4 @@
 USE_TCP=0
-BUFF_VIDEO_MAX=64
-BUFF_AUDIO_MAX=128
+BUFF_VIDEO_MAX=0
+BUFF_AUDIO_MAX=0
```

Comment:
Where these changes intended?

Reply:
Yes but it doesn't seem to make a difference. Reverted.